packages/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -245,15 +245,26 @@ const Calendar = WebexPlugin.extend({
   },
 
   /**
-   * Retrieves an array of meeting participants for the meeting id
-   * @param {String} id
+   * Retrieves an array of meeting participants for the meeting participantsUrl
+   * @param {String} participantsUrl
    * @returns {Promise} Resolves with an object of meeting participants
    */
-  getParticipants(id) {
+  getParticipants(participantsUrl) {
```

Comment:
I'm beginning to wonder what's the point of having these functions like getParticipants() and getNotesByUrl() in this internal-plugin-calendar. They don't do anything, they don't have any logic, just send a GET request and the caller has to know which special URL to pass to them so they seem a bit useless. Why do we have them? Are they going to be expanded later with more logic and this is just a start? Where are they called from?

Reply:
I checked code, when list  if the Calendar API doesn't send list of participants with the meeting object, need send getParticipants call for get meeting participants, i think happens for large meeting.

Reply:
From any space, click the Meetings tab (Messages, People, Content, Meetings) and the Calendar and Meeting content tabs should call getNotest.

Reply:
As origin design, call syncCalendar, return meeting object, if doesn't send meeting notes with the meeting object, need send getNotes call for get meeting notes, remove it in 2023.5.16 (PR: https://github.com/webex/webex-js-sdk/pull/2805)